### DISCUSSION TITLE
Internet connection issue when installing Bible modules

### DISCUSSION THREAD

**Discussion by ezala-web** [ezra-bible-app/ezra-bible-app#974] [General]
![Screenshot (3)](https://user-images.githubusercontent.com/122982550/215143614-598841b4-1ddf-497e-bd81-a55113db7b6c.png)

Instalei agora o software ezra bible, aqui no meu Pc não está aceitar instalar modulo online está aparecendo a mensagem que está na imagem mas aqui não tenho problema de Internet. Perante este problema gostaria que melhorasse o software para abilitar instalar modulos offline manualmente. Obrigado 

English translation (Google Translate):
I have now installed the ezra bible software, here on my PC it is not accepting to install the module online the message that is in the image is appearing but here I have no Internet problem. Faced with this problem, I would like you to improve the software to enable installing offline modules manually.

**Comment by tobias-klein** [comment]
Thank you for reporting this! I put the English text below your report based on Google translate (I don't understand Portuguese). 

Technically this could also happen when the connection is blocked by some firewall. Can you tell me what kind of internet connection you have been using when you faced this error? Was it a home internet connection or some kind public network?

**Comment by tobias-klein** [comment]
Hi @ezala-web!

A similar bug report was raised before, see #155.
Could you check the solution that was described there? (Raising the connection timeout)
see https://github.com/ezra-bible-app/ezra-bible-app/issues/155#issuecomment-750302401

**Comment by ezala-web** [comment]
Hello, I always use internet connection from home. When I try to download the modules, the software always shows that I am online.Em 29 de jan de 2023 17:26, Tobias Klein ***@***.***> escreveu:
Thank you for reporting this! I put the English text below your report based on Google translate (I don't understand Brazilian Portuguese).
Technically this could also happen when the connection is blocked by some firewall. Can you tell me what kind of internet connection you have been using when you faced this error? Was it a home internet connection or some kind public network?

**Comment by tobias-klein** [comment]
Hi @ezala-web
Could you try the workaround described in the linked issue above?
